packages/jspsych/src/JsPsych.ts

```diff
@@ -435,9 +440,9 @@ export class JsPsych {
 
     try {
       await Promise.all(
-        extensions.map((extension) =>
-          this.extensions[extension.type].initialize(extension.params || {})
-        )
+        extensions.map((extension) => {
+          this.extensions[extension.type.info.name].initialize(extension.params || {})
+        })
```

Comment:
You removed the implicit return of the arrow function and now it returns undefined. `Promise.all()` needs a list of promises, but gets a list of `undefined`s here :D
```suggestion
        extensions.map((extension) => 
          this.extensions[extension.type.info.name].initialize(extension.params || {})
        )
```

Reply:
Thanks! I added the `{}` because I was adding some other statements in this during testing. I didn't even realize that there was an implicit return!